Packs/Expanse/Integrations/Expanse/Expanse.py

```diff
@@ -0,0 +1,541 @@
+        "Admin": {
+            "Country": data['whois'][0]['admin']['country'],
+            "Email": data['whois'][0]['admin']['emailAddress'],
+            "Name": data['whois'][0]['admin']['name'],
+            "Phone": data['whois'][0]['admin']['phoneNumber']
+        },
+        "Registrant": {
+            "Country": data['whois'][0]['registrant']['country'],
+            "Email": data['whois'][0]['registrant']['emailAddress'],
+            "Name": data['whois'][0]['registrant']['name'],
+            "Phone": data['whois'][0]['registrant']['phoneNumber']
+        },
+        "WHOIS": {
+            "DomainStatus": data['whois'][0]['domainStatuses'],
+            "NameServers": data['whois'][0]['nameServers'],
+            "CreationDate": data['whois'][0]['creationDate'],
+            "UpdatedDate": data['whois'][0]['updatedDate'],
+            "ExpirationDate": data['whois'][0]['registryExpiryDate'],
+            "Registrant": {
+                "Email": data['whois'][0]['registrant']['emailAddress'],
+                "Name": data['whois'][0]['registrant']['name'],
+                "Phone": data['whois'][0]['registrant']['phoneNumber']
+            },
+            "Registrar": {
+                "Name": data['whois'][0]['registrar']['name'],
+                "AbuseEmail": data['whois'][0]['registrar']['abuseContactEmail'],
+                "AbusePhone": data['whois'][0]['registrar']['abuseContactPhone']
+            },
+            "Admin": {
+                "Name": data['whois'][0]['admin']['name'],
+                "Email": data['whois'][0]['admin']['emailAddress'],
+                "Phone": data['whois'][0]['admin']['phoneNumber']
+            }
+        },
+        "DateAdded": data['dateAdded'],
+        "FirstObserved": data['firstObserved'],
+        "LastObserved": data['lastObserved'],
+        "HasLinkedCloudResources": data['hasLinkedCloudResources'],
+        "SourceDomain": data['sourceDomain'],
+        "Tenant": data['tenant']['name'],
+        "BusinessUnits": [],
+        "DNSSEC": data['whois'][0]['dnssec'],
+        "RecentIPs": data['details']['recentIps'],
+        "CloudResources": data['details']['cloudResources'],
+        "LastSubdomainMetadata": data['lastSubdomainMetadata'],
+        "ServiceStatus": data['serviceStatus'],
+        "LastSampledIP": data['lastSampledIp']
+    }
+    for i in data['businessUnits']:
+        c['BusinessUnits'].append(i['name'])
+    return c
+
+
+'''MAIN FUNCTIONS'''
+
+
+def fetch_incidents_command():
+    ''' retrieve active exposures from Expanse API and create incidents '''
+    now = datetime.today()
+    today = datetime.strftime(now, "%Y-%m-%d")
+    yesterday = datetime.strftime(now - timedelta(days=1), "%Y-%m-%d")
+    last_run = demisto.getLastRun()
+    LOG(last_run)
+    start_date = yesterday
+    end_date = yesterday
+
+    try:
+        "start_time" in last_run
+        "complete_for_today" in last_run
```

Comment:
same here, not sure whats the reason for the try, except

Reply:
the first time the integration runs, these definitions will not be in the last_run dictionary, so the exception occurs and initializes the integration with default parameters.